bindings/nodejs/src/lib.rs

```diff
@@ -83,16 +83,61 @@ impl Operator {
     pub fn new(service_type: Scheme, options: Option<HashMap<String, String>>) -> Result<Self> {
         let ops = options.unwrap_or_default();
         match service_type {
+            Scheme::Azblob => Ok(Self(
+                opendal::Operator::new(opendal::services::Azblob::from_map(ops))
+                    .unwrap()
```

Comment:
Please raise an error as soon as possible whenever feasible. The `unwrap()` function must be used only when you confirm that the program is in an incorrect state.